File: Publisher_Subscriber_Nodes_tasks/psNode_task_ws/src/cpp_count_hundred/src/publisher.cpp
#include<ros/ros.h>
#include<std_msgs/Int32.h>

int main (int argc,char**argv)
{
    ros::init(argc,argv,"countPublisher");
    ros::NodeHandle nh;
    ros::Rate loop_rate(2);
    ros::Publisher pub = nh.advertise<std_msgs::Int32>("countTopic",1000);
    std_msgs::Int32 count;
    count.data = 100;
    while (ros::ok())
    {
        ROS_INFO("%d",count);
        pub.publish(count);
        ros::spinOnce();
        loop_rate.sleep();
        ++count.data;  
    }
    
    return 0;
}

File: Task5/psNode_task_ws/src/count_hundred/scripts/publisher.py
#!/usr/bin/env python3
import rospy
from std_msgs.msg import Int32

def talker():
    rospy.init_node("countPublisher_node")
    pub = rospy.Publisher("countTopic",Int32,queue_size=1)
    rate = rospy.Rate(2)
    count=100
    while not rospy.is_shutdown():
        rospy.loginfo(count)
        pub.publish(count)
        rate.sleep()
        count += 1





if __name__=='__main__' :
    try:
        talker()
    except rospy.ROSInterruptException :
        pass
    
